src/gs-shell.c

```diff
@@ -790,6 +790,10 @@ gs_shell_copy_dests_notify_cb (GsPluginLoader *plugin_loader,
 	GsShellPrivate *priv = gs_shell_get_instance_private (shell);
 	GsPage *page = GS_PAGE (g_hash_table_lookup (priv->pages, "overview"));
 
+	/* the overview page has not been loaded yet */
```

Comment:
I'd prefer a comment stating why this is important. Something like: `Don't reload if we're still loading, it may break the state/refresh`.
And good catch, BTW!

Reply:
Done.